Taskfile.yml

```diff
@@ -69,7 +92,7 @@ tasks:
       # before building the executables, so that we can embed this information
       # in the binaries.
       # To recover: delete local tag: git tag --delete tagname
-      - git tag -a {{.RELEASE_TAG}} -m ''
+      - git tag --annotate {{.RELEASE_TAG}} -m 'Release {{.RELEASE_TAG}}'
```

Comment:
The `release` target was already issuing an annotated tag. Here we simply make it even more explicit by using the long option name.